.github/workflows/frontend.yml

```diff
@@ -200,6 +200,37 @@ jobs:
           type: frontend
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  upload-codecov-bundle-stats:
```

Comment:
this adds an extra job which unfortunately pushes us over our concurrency limits that we're right up against (this will cause significant queueing for PRs) -- can you get this from some other job?  is this "bundle analysis" actually useful? (we have some other plugin which already does this and isn't noisy iirc)

Reply:
Are you able to point me to another workflow that you only run when changes are made to the frontend? I’m trying to address your previous comments about this running when changes unrelated to the frontend occur, and this seemed like an appropriate spot to put them.

Reply:
I'm confused what bundle size has to do with code coverage at all in the first place

I'm not familiar with the frontend workflows but you can ask in #discuss-frontend

Reply:
Alrighty, moved the bundle analysis stuff back over to the acceptance tests workflow, but added in the check to detect for FE file changes, and to only enable bundle analysis if that is true.